linux-headers/linux/kvm.h

```diff
@@ -772,8 +772,9 @@ struct kvm_ppc_smmu_info {
 #define KVM_CAP_PPC_ENABLE_HCALL 104
 #define KVM_CAP_CHECK_EXTENSION_VM 105
 
-/***** custom capability for symbolic execution support *****/
+/***** custom capabilities for symbolic execution support *****/
 #define KVM_CAP_MEM_RW 1021
+#define KVM_CAP_CPU_CLOCK_SCALE 1022
```

Comment:
This capability is not really symbex-related.
Explain in the code what this capabilities does (in an abstract sense).